Package.swift

```diff
@@ -1,28 +1,35 @@
+// swift-tools-version:4.0
 import PackageDescription
 
 let package = Package(
     name: "Moya",
+    products: [
+        .library(name: "Moya", targets: ["Moya"]),
+        .library(name: "ReactiveMoya", targets: ["ReactiveMoya"]),
+        .library(name: "RxMoya", targets: ["RxMoya"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "4.1.0")),
+        .package(url: "https://github.com/antitypical/Result.git", .upToNextMajor(from: "3.2.0")),
+        .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", .upToNextMajor(from: "2.0.0")),
```

Comment:
Should these be `.exact` versions as well?

Reply:
I guess we can change it to `.exact`, however it would kill the possibility of package updating using `swift package update`, right? The exact versions are defined in `Package.resolved`, which is equivalent to `Carthage.resolved`.